### ISSUE TITLE
Getting issue in starting backend server

### ISSUE THREAD

**Issue by khubaibalam2000** [GoogleChrome/chromium-dashboard#1246]
**Describe the bug**
Whenever I try to start backend server by "npm start", it shows me this error.
![npm start](https://user-images.githubusercontent.com/74249716/113049919-a94a5700-91bd-11eb-9b63-a720b982078c.PNG)

**To Reproduce**
1. Go to chromium-dashboard folder in your directory.
2. Start server by 'npm start'.
3. It will stuck at "Starting rollup", and then in few seconds, will give an error.
4. See error.  If applicable, add screenshots to help explain your problem.

**Expected behavior**
Server should be run properly.

**Comment by akgupta0777** [comment]
Duplicate of #1241

**Comment by akgupta0777** [comment]
@shivamag00 you can see the screenshot there is a windows icon there it simply means he is using windows OS

**Comment by shivamag00** [comment]
@akgupta0777, Yes from the screenshot, it seems that he must be using Windows but these days, some developers tend to use Windows theme even on other OS.  Hence, I prefer asking. 

However, on closer inspection of the screenshot, one can see directory paths which are similar to paths in Windows. So, it is Windows OS. Thank you for drawing my attention to the minute details inside the screenshot !!!

**Comment by shivamag00** [comment]
@khubaibalam2000, follow the suggestion in #1241 which says to use Linux or VM or WSL 2 in windows for avoiding errors that may result due to incompatibility between Linux and Windows.

**Comment by shivamag00** [comment]
Also, please let us know the versions of npm and node that you have.

**Comment by khubaibalam2000** [comment]
Thank you @akgupta0777  and @shivamag00 for your concerns. Okay, now I am going to try to with Linux. And as you ask about versions @shivamag00 , currently, my npm version is 6.14.11, and node version is 14.16.0.
